package/kedro_viz/server.py

```diff
@@ -120,9 +124,15 @@ def run_viz(port=None, line=None) -> None:
     if port in _VIZ_PROCESSES and _VIZ_PROCESSES[port].is_alive():
         _VIZ_PROCESSES[port].terminate()
 
+    if local_ns is not None and 'project_path' in local_ns:
```

Comment:
```suggestion
    if local_ns is not None and "project_path" in local_ns:
```
to stay consistent with using double quotes in our codebase